docs/Developers/Newcomers_Guide.rst

```diff
@@ -353,7 +353,8 @@ For more information about reviewing code, check out this `link <http://coala.io
 .. note::
 
     Reviewing code helps you to learn from other people's mistakes so you can
-    avoid making those same mistakes yourself in the future!
+    avoid making those same mistakes yourself in the future! Thus, you are
```

Comment:
`those` -> `the` makes more sense.

Reply:
imo it's fine, it refers to the mistakes from before. Also it would belong into an own commit to change this imo ;)
@shreyans800755 feel free to file an issue ;)